lib/puppet/util/windows/taskscheduler2_v1task.rb

```diff
@@ -2,6 +2,9 @@
 # It is designed to be a binary compatible API to puppet/util/windows/taskscheduler.rb but
 # will only surface the features used by the Puppet scheduledtask provider
 #
+require 'puppet/util/windows/taskscheduler2'
+require 'puppet/util/windows/taskscheduler' # Needed for the WIN32::ScheduledTask flag constants
+
 class Puppet::Util::Windows::TaskScheduler2V1Task
   # The error class raised if any task scheduler specific calls fail.
   class Error < Puppet::Util::Windows::Error; end
```

Comment:
I think there should be a `require 'puppet/util/windows'` here so that this is unnecessary...